Mathlib/Analysis/SpecialFunctions/Trigonometric/Bounds.lean

```diff
@@ -30,17 +30,20 @@ Here we prove the following:
 sin, cos, tan, angle
 -/
 
-
-noncomputable section
-
 open Set
 
 namespace Real
+variable {x : ℝ}
 
-open scoped Real
+lemma sin_le (hx : 0 ≤ x) : sin x ≤ x := by
```

Comment:
Isn't it simpler to deduce it from `sin_lt`?

Reply:
It is, but I was hoping my proof would make `sin_lt` itself easier to prove This seems to not be the case, since the derivative `1 - cos x` is not everywhere positive (and here we don't have the statement that says that if the derivative of a function is ae positive then the function is strictly monotone).